app/views/reviews/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="field">
+<%= form_for([@park, @review]) do |f| %>
+
+  <b><%= f.label :rating%></b>
+  <%= f.text_field :rating%>
+
+  <b><%= f.label :body, "Review"%></b>
+  <%= f.text_field :body%>
+
+  <%= f.submit "Edit Review" %>
+
+<% end %>
+
+<%= link_to "Back to #{@park.name}", park_path(@park) %>
```

Comment:
Consider moving this line out of this partial, so that you can also use it in new.html.erb